src/utils_request.js

```diff
@@ -180,14 +180,38 @@ export const requestAsBrowser = async (options = {}) => {
         };
     }
 
-    if (!gotScrapingOptions.isStream) return gotScraping(gotScrapingOptions);
+    if (!gotScrapingOptions.isStream) {
+        try {
+            return await gotScraping(gotScrapingOptions);
+        } catch (err) {
+            if (gotScrapingOptions.forceEncoded || gotScrapingOptions.url === encodeURI(gotScrapingOptions.url)) {
```

Comment:
`gotScrapingOptions` should only include valid `got-scraping` options, which the `forceEncoded` is not, so it should be kept separately or in `gotScrapingOptions.context`.

We should also probably add an option to turn this behavior off completely.